ui/app/components/pipeline-editor.hbs

```diff
@@ -36,14 +36,14 @@
 
                 <dd.Content class="bg-white rounded-md border border-gray-200 shadow-md text-sm">
                   <ul>
-                    {{!-- template-lint-disable no-invalid-interactive --}}
+                    {{! template-lint-disable no-invalid-interactive }}
```

Comment:
I wasn't aware of this formatting before, thank you for the update!

Reply:
I wasnt either! my prettier on save just updated it lol